index.html

```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+            </button>
+        <div class="project_container_1">
+            <div class="project_title">
+                <h3>E-commerce Website</h3>
+            </div>
+            <div class="project_description">
+                <h4>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. Has been the 
+                industry's standard.</h4>
+            </div>
+            <div class="project_image"></div>
+            <ul class="project_skills">
+                <li class="skill_container"><h4 class="skill">HTML</h4></li>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
+            </ul>
+            <button class="project_link"><h4 class="link_text">See project</h4></button>
+        </div>
+        <div class="project_container_2">
+            <div class="project_title">
+                <h3>University Portal</h3>
+            </div>
+            <div class="project_description">
+                <h4>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. Has been the 
+                industry's standard.</h4>
+            </div>
+            <div class="project_image"></div>
+            <ul class="project_skills">
+                <li class="skill_container"><h4 class="skill">HTML</h4></li>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
+            </ul>
+            <button class="project_link"><h4 class="link_text">See project</h4></button>
+        </div>
+        <div class="project_container_3">
+            <div class="project_title">
+                <h3>Company Website</h3>
+            </div>
+            <div class="project_description">
+                <h4>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. Has been the 
+                industry's standard.</h4>
+            </div>
+            <div class="project_image"></div>
+            <ul class="project_skills">
+                <li class="skill_container"><h4 class="skill">HTML</h4></li>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
+            </ul>
+            <button class="project_link"><h4 class="link_text">See project</h4></button>
+        </div>
+        <div class="project_container_4">
+            <div class="project_title">
+                <h3>Business Website</h3>
+            </div>
+            <div class="project_description">
+                <h4>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. Has been the 
+                industry's standard.</h4>
```

Comment:
As I mentioned above, kindly, I suggest you use a `p` tag instead of a `h4` tag for the description, please, note that this is for good semantics and best practices. 😄